Library/Homebrew/tap.rb

```diff
@@ -526,13 +526,11 @@ def read_or_set_private_config
 
 # A specialized {Tap} class for the core formulae
 class CoreTap < Tap
-  if OS.mac?
-    def default_remote
-      "https://github.com/Homebrew/homebrew-core"
-    end
-  else
-    def default_remote
-      "https://github.com/Linuxbrew/homebrew-core"
+  def default_remote
+    if OS.mac? || ENV["$HOMEBREW_FORCE_HOMEBREW_ORG"]
```

Comment:
Was fixed in https://github.com/Homebrew/brew/commit/3589e24df03adc074081549229d8cc244643eee4 (after an hour of debugging...)